index.html

```diff
@@ -194,49 +198,228 @@ <h2>
             <code><dfn data-dfn-for="getDisplayMedia">getDisplayMedia</dfn></code>
           </dt>
           <dd>
+            <p>Prompts the user for permission to live-capture their display.</p>
             <p>
-              This method operates identically to <code><a>getUserMedia</a></code>, except that it
-              acquires media from display devices, and the <code>constraints</code> argument does
-              not accept <code><a href=
-              "https://w3c.github.io/mediacapture-main/#idl-def-mediatrackconstraints">MediaTrackConstraints</a></code>
-              values. This prevents an application from influencing the selection of sources, see
-              <a href="#constraints"></a> for details.
+              This method is similar to <code><a>getUserMedia</a></code>, except that it
+              acquires media from one display device chosen by the end-user each time.
```

Comment:
"one display device" sounds like single-monitor only?

Reply:
We might add a definition of "a display device", something like "In this context, a display device consists of all the display surfaces that are presented to the user to choose from."

Reply:
I think this is covered by *"Multiple monitors might also be aggregated into as a single logical monitor."*